virtualmachine/vsphere/util.go

```diff
@@ -108,6 +109,78 @@ var readAll = func(r io.Reader) ([]byte, error) {
 	return ioutil.ReadAll(r)
 }
 
+var extractOva = func(basePath string, body io.Reader) (string, error) {
+        fmt.Printf("Extracting ova to path: %s\n", basePath)
```

Comment:
I don't think we should have fmt.Printf() logging in a library that will be used by others, but when this goes upstream can get their input